### ISSUE TITLE
Add usage intructions in the README

### ISSUE THREAD

**Issue by gabrielaraujof** [gabrielaraujof/shadow-science#12]
For allow the ease use of the library, would be great to document on README how to use the library on any project.

**Comment by HarryJamesRobson** [comment]
I'll be happy to do this if you want to assign me.

**Comment by mihirs16** [comment]
hey, I was thinking of working on this, but I see that the index.html couldn't import and use the style.scss, am I missing something?

**Comment by vromao** [comment]
> hey, I was thinking of working on this, but I see that the index.html couldn't import and use the style.scss, am I missing something?

You **don't see** styles import in index.html because **styles.scss** is importing in **index.js**. Who handles with imports (generating bundles) here is **Webpack** (if you see **webpack.config.js** you will understand what's going on. About participate in this projetct, you'll need the aprove of @gabrielaraujof

**Comment by gabrielaraujof** [comment]
Yes, @mihirs16 of course you can take this one. For the moment we don't have a CSS version of the lib that could be easily imported on any HTML. Right now the only way to use it would be importing on Sass (the way you see on demo). However, we plan to have a CSS version asap.

**Comment by mihirs16** [comment]
So I guess the readme, for now, should have how to transpile the SASS to CSS and then import it into HTML?

**Comment by vromao** [comment]
> So I guess the readme, for now, should have how to transpile the SASS to CSS and then import it into HTML?

I think the readme should have how to use the lib. Diferences betwen attributes. In general how the lib works and its configurations and variations. It's not neccessary explain how to transpile in readme. I guess we'll open a PR with css version available in github project root

**Comment by mihirs16** [comment]
@vromao so that includes the functions and variables in the src file?

**Comment by gabrielaraujof** [comment]
Right now, it's enough to document these topics, in order:

- Usage
  - Importing on Sass
  - Adding attributes
- Shadow variations
  - Direction
  - Elevation

All topics should include at least one code example. And that's it, nothing more than that.

**Comment by vromao** [comment]
> @vromao so that includes the functions and variables in the src file? or just the css being generated for now?

I agree whit topics @gabrielaraujof mentioned above. Importing on Sass if user wants to use that way (but not how to transpile), how attributes works with the lib and HTML and how they can be used to achive shadow viriations o/
